project/Build.scala

```diff
@@ -13,14 +13,11 @@ import scoverage.ScoverageSbtPlugin
 
 object Scrooge extends Build {
   val branch = Process("git" :: "rev-parse" :: "--abbrev-ref" :: "HEAD" :: Nil).!!.trim
-  val suffix = if (branch == "master") "" else "-SNAPSHOT"
```

Comment:
Sorry, I'll put it back in!
Maybe there should be a "snapshot" mode in the build, like
`sbt snapshot compile`? Then it wouldn't be tied to the git branch name. E.g. in a setup with multiple remotes, "eirslett-master" is my local name for the eirslett/master branch, while "twitter-master" is my local name for the twitter/master branch.